apps/nextjs-website/react-components/components/HeroCounter/HeroCounter.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Box, Typography, Link } from '@mui/material';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import { isJSX } from '../../types/common/Common.types';
+import ContainerRC from '../common/ContainerRC';
+import { HeroCounterProps } from '@react-components/types/HeroCounter/HeroCounter.types';
+import { BackgroundColor, TextColor } from '../common/Common.helpers';
+import { useTheme } from '@mui/material/styles';
+
+const HeroCounter = (props: HeroCounterProps) => {
+  const {
+    background,
+    theme = 'dark',
+    counterNumber = 0,
+    counterText,
+    title,
+    subtitle,
+    linkText,
+    linkUrl,
+  } = props;
+
+  const backgroundColor = BackgroundColor(theme);
+  const textColor = TextColor(theme);
+
+  const BackgroundImage = isJSX(background) ? (
+    background
+  ) : (
+    <Box
+      role='presentation'
+      sx={{
+        px: 4,
+        position: 'absolute',
+        inset: 0,
+        zIndex: -10,
+        height: '100%',
+        width: '100%',
+        objectFit: 'cover',
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+        backgroundImage: `url(${background ?? ''})`,
+      }}
+    />
+  );
```

Comment:
```suggestion
  const BackgroundImage = (
    <Box
      role='presentation'
      sx={{
        px: 4,
        position: 'absolute',
        inset: 0,
        zIndex: -10,
        height: '100%',
        width: '100%',
        objectFit: 'cover',
        backgroundSize: 'cover',
        backgroundPosition: 'center',
        backgroundImage: `url(${background ?? ''})`,
      }}
    />
  );
```
`background` is always going to be a string.